specification/turn/src/test/java/org/kaazing/specification/turn/PeerConnectionIT.java

```diff
@@ -115,5 +116,76 @@ public void shouldSuccessfullySendDataMessage() throws Exception {
         k3po.finish();
     }
 
+    /**
+     * See <a href="https://tools.ietf.org/html/rfc5766">RFC 5766: Turn Protocol</a>.
+     */
+    @Test
+    @Ignore ("Please see issue #731: https://github.com/kaazing/tickets/issues/731")
+    @Specification({
+            "incorrect.channel.data.message.reserved.channel.number/request",
+            "incorrect.channel.data.message.reserved.channel.number/response" })
+    public void shouldFailSendingDataMessageReservedChannelNumbber() throws Exception {
+        k3po.finish();
+    }
+
+    /**
+     * See <a href="https://tools.ietf.org/html/rfc5766">RFC 5766: Turn Protocol</a>.
+     */
+    @Test
+    @Ignore ("Please see issue #731: https://github.com/kaazing/tickets/issues/731")
+    @Specification({
+            "incorrect.channel.data.message.short.lenght/request",
+            "incorrect.channel.data.message.short.lenght/response" })
+    public void shouldFailSendingDataMessageTooShortLength() throws Exception {
+        k3po.finish();
+    }
+
+    /**
+     * See <a href="https://tools.ietf.org/html/rfc5766">RFC 5766: Turn Protocol</a>.
+     */
+    @Test
+    @Ignore ("Please see issue #731: https://github.com/kaazing/tickets/issues/731")
+    @Specification({
+            "incorrect.channel.data.message.wrong.channel.number/request",
+            "incorrect.channel.data.message.wrong.channel.number/response" })
+    public void shouldFailSendingDataMessageWrongChannelNumber() throws Exception {
+        k3po.finish();
+    }
+
+    /**
+     * See <a href="https://tools.ietf.org/html/rfc5766">RFC 5766: Turn Protocol</a>.
+     */
+    @Test
+    @Ignore ("Please see issue #733: https://github.com/kaazing/tickets/issues/733")
+    @Specification({
+            "channel.data.message.without.create.permissions/request",
+            "channel.data.message.without.create.permissions/response" })
+    public void shouldFailSendingDataMessageWithoutPermissions() throws Exception {
+        k3po.finish();
+    }
+
+    /**
+     * See <a href="https://tools.ietf.org/html/rfc5766">RFC 5766: Turn Protocol</a>.
+     */
+    @Test
+    @Ignore ("Please see issue #733: https://github.com/kaazing/tickets/issues/733")
```

Comment:
I think I see.  Write tests like this as if you are testing a server.  I.e.  Bytes on the wire is what you code in scripts and then the reaction would not be in script.  i.e.
```
Bytes on the wire                                              Code API

--- don't read this request   --->   [ server ]  
--- read this request         --->                  ---> forwards second request in code 
```

Then when we test the proxy we do 

```
Bytes on the wire/k3po                                         Bytes on the wire / k3po

script with 
--- don't read this request   --->   [ server ]  
--- read this request         --->                  ---> script for normal request
```

Hope this makes sense, if not feel free to ping me